Mathlib/Topology/Algebra/InfiniteSum/Group.lean

```diff
@@ -1,10 +1,11 @@
 /-
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Johannes Hölzl
+Authors: Johannes Hölzl, Mitchell Lee
 -/
 import Mathlib.Topology.Algebra.InfiniteSum.Basic
 import Mathlib.Topology.Algebra.UniformGroup
+import Mathlib.Topology.Algebra.GroupCompletion
```

Comment:
Please move the new results to a new file (`InfiniteSum/GroupCompletion`?) unless this transitively imports very few new files.